notebooks/a00_introduction_to_geomstats.ipynb

```diff
@@ -0,0 +1,290 @@
+{
```

Comment:
<p>Ah! This makes me realize that we have not made the hierarchy for the Connection / Riemannian Metric classes. I will try to get this to you soon, this is actually a separate hierarchy as the one for the manifolds.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1304/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>RiemannianMetric does inherit from Connection as it is a more general tool, and this can be justified by defining the Levi-Civita connection of a Riemannian Metric</p>

Reply:
<p>Thanks! I have edited this image to show that RiemannianMetric inherits from Connection</p>